### ISSUE TITLE
Normals' static conditioning values

### ISSUE THREAD

**Issue by matan-feldman** [philgras/neural-head-avatars#20]
Hey, 

I was wondering what’s the reason for using constant `_static_conditioning` values in [`SirenNormalEncoder`](https://github.com/philgras/neural-head-avatars/blob/473457eef83c9ee26f316451e31c2aa01a74603c/nha/models/siren_mlps.py#L193)?
Why isn’t this attribute just a zeros tensor/ being calculated based on other parameters?

 And more precisely, what do these specific constant values represent? Are they only valid for specific configurations?

Thanks a lot in advance 🙏 

```
self.register_buffer("_static_condition",  # 1 x 100
                     torch.tensor([[-1.10247604, -1.42804892, 0.87488722, 1.15597923, -0.57568966,
                                    -0.04965701, 0.07089813, -2.21527982, 0.13931708, 0.30833226,
                                    1.7272208, 0.40821049, -0.86179693, 0.49263112, 1.1502004,
                                    1.01742589, -0.91245066, -0.43667411, -0.1643309, 1.37222254,
                                    -0.40959126, -1.26752158, 1.84200903, -0.42769857, -1.72281049,
                                    1.54676358, 0.70155848, 1.32231672, -1.55473343, -0.64930216,
                                    1.33731289, 0.68014974, -0.87769585, -1.13550678, -0.24108198,
                                    -0.56245497, -0.26652164, -1.67956008, -0.4283457, 1.26249083,
                                    0.40890499, 0.40677053, -0.44340142, 1.25217541, 2.13999611,
                                    -1.17023326, -1.10413861, -0.30643598, -0.17989996, 0.7573133,
                                    -0.09529626, -1.67288201, -0.56437952, 1.14097756, 0.43888664,
                                    0.73405574, -0.99273702, 0.15076723, 0.65683655, 1.44445144,
                                    -0.32217313, 0.50703575, -0.98423476, -1.13127301, -0.65218117,
                                    0.18305708, 0.26766889, 0.30279705, -2.3601944, -1.4982377,
                                    1.86408925, 1.11687454, -0.2839407, 1.7602057, 0.4117718,
                                    0.34926186, 0.66022188, -0.55205758, 0.03506068, -0.01726678,
                                    -0.18771338, 0.03045248, -0.04338471, 0.01669747, -0.84164925,
                                    0.95326186, 0.52841106, 1.71889631, -1.26329448, -0.19737745,
                                    -1.10392851, -0.39721072, -0.42183207, 1.73825699, -1.24009311,
                                    -0.84449652, 0.43659571, -0.00715666, 0.35093695, 1.20645271]]))
```

**Comment by malteprinzler** [comment]
Thank you very much for your interest in our work. These parameters are just the result of a call to torch.randn(). They were set as fixed for reproducibility reasons. We did not want to make them all zeros bc. this would remove all impact of the weight parameters in the convolutional network. Overall, we did not experiment much with this part of the network so there is no strong meaning behind this architecture choice.
